docs/topics/Permissions.md

```diff
@@ -179,17 +179,18 @@ Roles represent a set of permissions attached to a group of users. Roles have un
-| name        | string                                                                       | role name                                        |
-| color       | integer                                                                      | integer representation of hexadecimal color code |
-| hoist       | boolean                                                                      | if this role is pinned in the user listing       |
-| position    | integer                                                                      | position of this role                            |
-| permissions | string                                                                       | permission bit set                               |
-| managed     | boolean                                                                      | whether this role is managed by an integration   |
-| mentionable | boolean                                                                      | whether this role is mentionable                 |
-| tags?       | [role tags](#DOCS_TOPICS_PERMISSIONS/role-object-role-tags-structure) object | the tags this role has                           |
+| Field       | Type                                                                         | Description                                       |
+| ----------- | ---------------------------------------------------------------------------- | ------------------------------------------------- |
+| id          | snowflake                                                                    | role id                                           |
+| name        | string                                                                       | role name                                         |
+| color       | integer                                                                      | integer representation of hexadecimal color code  |
+| hoist       | boolean                                                                      | if this role is pinned in the user listing        |
+| icon        | ?string                                                                      | role [icon hash](#DOCS_REFERENCE/image-formatting)|
+| position    | integer                                                                      | position of this role                             |
+| permissions | string                                                                       | permission bit set                                |
+| managed     | boolean                                                                      | whether this role is managed by an integration    |
+| mentionable | boolean                                                                      | whether this role is mentionable                  |
+| tags?       | [role tags](#DOCS_TOPICS_PERMISSIONS/role-object-role-tags-structure) object | the tags this role has                            |
```

Comment:
there's an `unicode_emoji` field, at least I got it on a `GUILD_ROLE_UPDATE`.

Reply:
This seems to be returned by the rest api as well.

Reply:
but you can use guild emojis too.
Or is that only set when using unicode emoji